src/components/policy/policy_regular/src/policy_helper.cc

```diff
@@ -1022,7 +1022,7 @@ void FunctionalGroupInserter::operator()(const StringsValueType& group_name) {
 
 void FillFunctionalGroupPermissions(
     FunctionalGroupIDs& ids,
-    const FunctionalGroupNames& names,
+    FunctionalGroupNames& names,
```

Comment:
Why `const` was added in previous commit and then is deleted in this commit? Are this changes needed?